memcached/build.gradle

```diff
@@ -0,0 +1,8 @@
+dependencies {
+  implementation(libs.spymemcached)
+  implementation(libs.commonsLang3)
+  implementation project(':framework')
+  testImplementation project(':framework').sourceSets.test.compileClasspath
+}
+
+apply from: rootProject.file('gradle/deploy.gradle')
```

Comment:
No new line at end of file.

`.editorconfig` is not active yet?

Reply:
what i'd do: i just run formatter once in a while on `main`.

but I'll fix it for you. (prolly will never develop into a habit though)

Reply:
Fixed!